packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -84,10 +86,10 @@ contract DeploySuperchainInput {
 
     // Load the input from a TOML file.
     function loadInputFile(string memory _infile) public {
-        _infile;
-        Input memory parsedInput;
+        string memory toml = vm.readFile(_infile);
+        bytes memory data = vm.parseToml(toml);
+        Input memory parsedInput = abi.decode(data, (Input));
```

Comment:
In a future PR we probably should refactor this to read each input indivudually then craft the `Input` struct at the end. This would allow us to configure defaults and more easily reuse code when a WIP feature changes the `Input` struct. But for this PR's scope this definitely works